share/completions/systemctl.fish

```diff
@@ -1,8 +1,15 @@
+set -l systemd_version (systemctl --version | string match "systemd*" | string replace -r "\D*(\d+)"  '$1')
```

Comment:
If there is no `systemctl` command, you're going to get a lot of errors. This will go away when we stop loading completion scripts for non-existing commands.

```
/usr/local/Cellar/fish/HEAD-47ad707/share/fish/completions/systemctl.fish (line 1): systemctl --version | string match "systemd*" | string replace -r "\D*(\d+)"  '$1'
                                                                                    ^
in command substitution
	called on line -1 of file /usr/local/Cellar/fish/HEAD-47ad707/share/fish/completions/systemctl.fish

from sourcing file /usr/local/Cellar/fish/HEAD-47ad707/share/fish/completions/systemctl.fish
	called on standard input

in command substitution
	called on standard input

test: Missing argument at index 2
test: Missing argument at index 2
test: Missing argument at index 2
```